src/applications/check-in/day-of/pages/ArrivedAtFacility.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { recordEvent } from '@department-of-veterans-affairs/platform-monitoring/exports';
+import { useFormRouting } from '../../hooks/useFormRouting';
+import { recordAnswer } from '../../actions/universal';
+import { createAnalyticsSlug } from '../../utils/analytics';
+import { makeSelectCurrentContext, makeSelectApp } from '../../selectors';
+
+import Wrapper from '../../components/layout/Wrapper';
+
+import { APP_NAMES } from '../../utils/appConstants';
+
+const ArrivedAtFacility = props => {
+  const { router } = props;
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const { goToNextPage, getCurrentPageFromRouter } = useFormRouting(router);
+  const currentPage = getCurrentPageFromRouter();
+
+  const pageType = 'arrived-at-facility';
+  const testID = 'arrived-page';
+  const bodyText = (
+    <>
+      <p>{t('we-ask-this-question-because-staff-can-call-you')}</p>
+    </>
+  );
+
+  const selectCurrentContext = useMemo(makeSelectCurrentContext, []);
+  const { setECheckinStartedCalled } = useSelector(selectCurrentContext);
+  const selectApp = useMemo(makeSelectApp, []);
+  const { app } = useSelector(selectApp);
+
+  const onClick = event => {
+    const answer = event.target.attributes.value.value;
+    recordEvent({
+      event: createAnalyticsSlug(
+        `${answer}-to-${pageType}${
+          setECheckinStartedCalled || app !== APP_NAMES.CHECK_IN ? '' : '-45MR'
+        }-clicked`,
+        'nav',
+        app,
+      ),
+    });
+  };
+
+  const yesAction = event => {
+    recordEvent({
+      event: createAnalyticsSlug('arrived-at-facility-yes'),
+      fromPage: currentPage,
+    });
+    dispatch(recordAnswer({ [pageType]: 'yes' }));
+    onClick(event);
+    goToNextPage();
+  };
+
+  const noAction = event => {
+    recordEvent({
```

Comment:
remove this record event since you have the other

Reply:
Not sure what you mean, since one is the "no" event', one is the "yes" event, and the other is the "45mr" event which is recorded for both